tests/test_models/test_state.py

```diff
@@ -100,6 +100,6 @@ def test_to_dict_values(self):
 
     def test_str(self):
         """test that the str method has the correct output"""
-        state = State()
-        string = "[State] ({}) {}".format(state.id, state.__dict__)
-        self.assertEqual(string, str(state))
+        s = State()
+        string = "[State] ({}) {}".format(s.id, s.__dict__)
+        self.assertEqual(string, str(s))
```

Comment:
Many of the same issues as City. Rinse and repeat.

Reply:
You got it!